test/image-blueprints/layer5-bootc/group2/rhel94-bootc-source-optionals.containerfile

```diff
@@ -1,31 +1,49 @@
 FROM localhost/rhel94-bootc-source:latest
 
 # Build arguments
-ARG PREVIOUS_MINOR_VERSION=15
 ARG USHIFT_RPM_REPO_NAME=microshift-local
 ARG USHIFT_RPM_REPO_PATH=/tmp/$USHIFT_RPM_REPO_NAME
 
 # Copy the MicroShift repository contents
 COPY $USHIFT_RPM_REPO_NAME $USHIFT_RPM_REPO_PATH
 
-# MicroShift local RPM repository
-RUN printf "\
+# Add the following repositories, using host entitlements for SSL keys:
+# - MicroShift local RPM repository
+# - Fast Datapath repository for MicroShift dependencies
+# - OpenShift previous minor version repository for MicroShift dependencies
+RUN sslkey=$(find /etc/pki/entitlement-host/ -type f -name "*-key.pem" -print -quit) && \
```

Comment:
This run command is one the longer end of what I usually see.  Could we extract this to it's own script file to be `ADD`'d during build time?